### ISSUE TITLE
poetry shell modifies $TERM env variable within TMUX

### ISSUE THREAD

**Issue by owittek** [python-poetry/poetry#7861]
- **Poetry version**:  Poetry (version 1.4.2)
- **Python version**:  3.11.3
- **OS version and name**:  macOS 13.3.1

- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) stable Poetry version, installed using a recommended method.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have consulted the [FAQ](https://python-poetry.org/docs/faq/) and [blog](https://python-poetry.org/blog/) for any relevant entries or release notes.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option) and have included the output below.

## Issue

Hey, I'm an avid tmux user and love poetry so far, thanks for your work!

The issue I am experiencing is that when I run `poetry shell` my `$TERM` env variable is being changed from `screen-256color` to `tmux-256color` which breaks my terminal colors. Manually setting it back to `screen-256color` works fine. I'm running zsh 5.9 as my shell of choice.

**Comment by dimbleby** [comment]
Not a `zsh` user and no idea what the implications of `emulate` are, but you can read all about its introduction at #5795

I avoid `poetry shell` altogether, just activate the virtual environment yourself as though `poetry shell` did not exist

**Comment by owittek** [comment]
> Not a `zsh` user and no idea what the implications of `emulate` are, but you can read all about its introduction at #5795
> 
> I avoid `poetry shell` altogether, just activate the virtual environment yourself as though `poetry shell` did not exist

Yeah I do the same thing to prevent this issue but I still wanted to report this so it can be fixed eventually if it's not just an issue on my side.

**Comment by dimbleby** [comment]
if you know how to fix it, submit a merge request!

(personally I'd like to see `poetry shell` removed altogether, anyway I wouldn't expect that waiting for someone else to fix such things is likely to be a successful strategy)

**Comment by owittek** [comment]
I've actually skimmed the code for `poetry shell` but I didn't for where the `emulate` command is being invoked. I'll try to confirm my suspicion later on & if it's true I'll also look more in-depth into the source code.
